letsencrypt/tests/reverter_test.py

```diff
@@ -405,30 +442,35 @@ def setup_test_files():
 
 def get_save_notes(dire):
     """Read save notes"""
-    return read_in(os.path.join(dire, 'CHANGES_SINCE'))
+    return read_in(os.path.join(dire, "CHANGES_SINCE"))
 
 
 def get_filepaths(dire):
     """Get Filepaths"""
-    return read_in(os.path.join(dire, 'FILEPATHS'))
+    return read_in(os.path.join(dire, "FILEPATHS"))
 
 
 def get_new_files(dire):
     """Get new files."""
-    return read_in(os.path.join(dire, 'NEW_FILES')).splitlines()
+    return read_in(os.path.join(dire, "NEW_FILES")).splitlines()
+
+
+def get_undo_commands(dire):
+    """Get new files."""
+    return csv.reader(open(os.path.join(dire, "COMMANDS")))
```

Comment:
This causes a leak as file is never closed. Possible fix:
```python
import contextlib

@contextlib.contextmanager
def get_undo_commands(dire):
    with open(os.path.join(dire, "COMMANDS")) as commands_file:
        yield csv.reader(commands_file)

with get_undo_commands(path) as reader:
    ...
```

or just
```python
def get_undo_commands(dire):
    with open(...) as commands_file:
        return list(csv.reader(...))
```

You may want to update the documentation/name though.